### ISSUE TITLE
NMAMapTrackingOrientation in android

### ISSUE THREAD

**Issue by bluesnow4425** [heremaps/here-android-sdk-examples#268]
hi, 
we're building app on both ios and android version, however we've found there're many differences between the navigation camera tracking part.

As 2 open issues i posted before, i found if i can make the map's orientation same with the current heading toward the top of the screen, the problem might solve much.

I know from our ios colleague that there's a method they can use to let the map automatically turns: 
https://developer.here.com/documentation/ios-premium/api_reference_jazzy/Enums/NMAMapTrackingOrientation.html#%2Fc:@E@NMAMapTrackingOrientation@NMAMapTrackingOrientationDynamic

Is the method available in android as well?

**Comment by starand** [comment]
There is RoadView subclass in NavgationManager. It contains setOrientation(orientation) method.
See https://developer.here.com/documentation/android-premium/api_reference_java/com/here/android/mpa/guidance/NavigationManager.RoadView.htm

Is it enough?

**Comment by bluesnow4425** [comment]
thanks for the reference. However it doesn't work as expected. 
I print the log and see roadview using Dynamic for orientation as a default. So for long i've been using dynamic already, but I was expecting **the map rotate as the mobile heads to different direction** and it didn't.

Is there any suggestion for expected behavior?

**Comment by bluesnow4425** [comment]
also, i tried to set it as NorthUp but in vain.

`NavigationManager.setMapUpdateMode(NavigationManager.MapUpdateMode.ROADVIEW);`
		`NavigationManager.getRoadView().setOrientation(NavigationManager.RoadView.Orientation.NORTH_UP);`

`Log.d("orientation", navigationManager.getRoadView().getOrientation().name());`

and i see log prints 

> D/orientation: DYNAMIC

which seems not working.

**Comment by Merlin1stHere** [comment]
@bluesnow4425 NMAMapTrackingOrientation and NavigationManager.RoadView.Orientation simply tells the NavigationManager whether it should rotate the map along the current road or not. The SDK does not use the accelerometer and compass of the device.

About result of the navigationManager.getRoadView().getOrientation() function:
I checked it and got the expected result. If I set the orientation to NORTH_UP, I get the same value. Make sure that setMapUpdateMode and setOrientation are called after starting navigation.
